packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergBridgeJS2Parent.java

```diff
@@ -166,14 +163,13 @@ void gutenbergDidRequestUnsupportedBlockFallback(ReplaceUnsupportedBlockCallback
 
     void onShowXpostSuggestions(Consumer<String> onResult);
 
-    void requestMediaFilesEditorLoad(ReplaceMediaFilesEditedBlockCallback replaceMediaFilesEditedBlockCallback,
-                                                     ReadableArray mediaFiles,
-                                                     String blockId
-    );
+    void requestMediaFilesEditorLoad(ReadableArray mediaFiles, String blockId);
```

Comment:
unrelated to this PR, removed the ReplaceMediaFilesBlockCallback as it was not being used

Reply:
makes sense! Thanks!